positron/components/PositronCLH.manifest

```diff
@@ -1,3 +1,3 @@
 component {dd20d954-f2a9-11e5-aeae-782bcb9e2c3f} PositronCLH.js
 contract @mozilla.org/positron/clh;1 {dd20d954-f2a9-11e5-aeae-782bcb9e2c3f}
-category command-line-handler x-positron @mozilla.org/positron/clh;1
+category command-line-handler a-positron @mozilla.org/positron/clh;1
```

Comment:
It seems fine to have the application CLH run early like this, as far as I can tell. Firefox actually break theirs into two, one at `m-browser` and another at `x-default`. It seems like people are just making up names as needed to get the ordering right...